spec/integration/alerts_spec.rb

```diff
@@ -623,6 +655,17 @@ module Hawkular::Alerts::RSpec
       expect(events.size).to be >= 12
     end
 
+    it 'Should list events for multiple tenants' do
+      client = Hawkular::Alerts::Client.new(ALERTS_BASE, creds, options)
+
+      events = client.events(criteria: { 'thin' => true }, tenants: [:hawkular, :foo])
+
+      expect(events).to_not be_nil
+      expect(events.size).to be >= 24
+      expect(events.count { |e| e.tenantId == 'hawkular' }).to be >= 12
+      expect(events.count { |e| e.tenantId == 'foo' }).to be >= 12
```

Comment:
Hmm... Had to filter by `eventType`, because it is returning alerts. Don't know why, because alerts cleanup should already be made. Additionally, for `hawkular` tenant I couldn't change `>=` to `eq`. For hawkular tenant, some other events are being returned. May be because Hawkular is adding events to `hawkular` tenant to monitor itself?